titus-server-master/src/main/java/com/netflix/titus/master/mesos/kubeapiserver/direct/taint/DefaultTaintTolerationFactory.java

```diff
@@ -56,23 +49,13 @@ public DefaultTaintTolerationFactory(DirectKubeConfiguration configuration,
         tolerations.add(Tolerations.TOLERATION_VIRTUAL_KUBLET);
         tolerations.add(Tolerations.TOLERATION_KUBE_SCHEDULER);
 
-        tolerations.add(resolveTierToleration(job));
```

Comment:
is this backwards compatible? I think we may need to keep double writing for now until all other components are aligned

Reply:
I think you are right. It would be too risky to assume we can make all updates in one shot.